content/issues/2023-04-05-copr-outage.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Upgrade of Copr servers"
+date: "2023-04-05T12:00:00+00:00"
+affected:
+  - Copr
+resolved: false
```

Comment:
https://github.com/packit/status#in-more-detail says "remove `resolved:` completely to make an announcement"

Reply:
ah, I missed that, thanks!